tests/integrations/config/cors.py

```diff
@@ -0,0 +1,15 @@
+"""Cross-Origin Resource Sharing (CORS) config"""
+
+PATHS = ["api/*", "sanctum/csrf-cookie", "server1.com"]
+
+ALLOWED_METHODS = ["*"]
+
+ALLOWED_ORIGINS = ["*"]
```

Comment:
I think we need to have these configs more similar to our other configs. 

Something like:

```
CORS = {
   "paths": ["api/*", "sanctum/csrf-cookie", "server1.com"],
   "allowed_methods": ["*"],
   ....
   
}
```

Reply:
The rest of the PR looks good but still not sold on this.

> The issue is with this is that it's already namespaced with cors through the config file name...so accessing it later will result in doing `config("cors.cors.paths")`....

You have a valid point here. Honestly it looks better than this config file though. This looks very M3 to me.

Just throwing some idea out there but do we have like a `security.py` file so at least it would be `config("security.cors.paths")`? And we can put other security settings in there either now or later

Reply:
Hmm we don't have it yet. But we could add it, `cors` was already new so we could change it to `security` why not, i like it